JuiceMaker/JuiceMaker/Controller/StockManagerViewController.swift

```diff
@@ -38,11 +56,34 @@ final class StockManagerViewController: UIViewController {
             .pineapple: pineappleCountLabel,
             .mango: mangoCountLabel
         ]
+        
+        fruitSteppersDictionary = [
+            strawberryStepper: .strawberry,
+            bananaStepper: .banana,
+            kiwiStepper: .kiwi,
+            pineappleStepper: .pineapple,
+            mangoStepper: .mango
+        ]
     }
 
     private func configureFruitCountLabels() {
         for fruit in Fruit.allCases {
             fruitLabelsDictionary[fruit]?.text = fruitStore.getStockCountToString(of: fruit)
         }
     }
+    
+    private func configureStepperDefaultValue() {
+        let _ = fruitSteppersDictionary.map { (stepper, fruit) in
+            guard let count = fruitStore.getStockCountToString(of: fruit),
+                  let countToDouble = Double(count)
+            else { return }
+            stepper.value = countToDouble
+        }
```

Comment:
`countToDouble`

[API Design Guidelines](https://www.swift.org/documentation/api-design-guidelines/)에서 다음과 같은 문구가 있습니다.

> Name variables, parameters, and associated types according to their roles, rather than their type constraints.
> : 타입 제약 조건이 아닌 **역할**에 따라 변수, 매개 변수 및 관련 유형의 이름을 지정합니다.
>
> ex) var string = "Hello" ---> var greeting = "Hello"

이렇듯 특정 타입을 명시하기 보다는 해당 프로퍼티의 역할이나 의미로 명명하는게 좋겠습니다!

Reply:
> 고차함수 `map`을 사용하셨네요! `map`은 어떤 동작을 하는걸까요?
> 
> 현재 반환되는 값을 사용하지 않아 `_`로 처리하신 것 같은데 값을 반환하지 않는 방식으로 처리하는건 어떨까요?

고차함수 map  같은 경우에는 항상 배열을 리턴해주기 때문에 반환값이 사용될 일이 없어서 for 문을 사용하여 처리하도록 수정하였습니다 !

Reply:
> `configureStepperDefaultValue` 현재 재고로 설정하는 것이니 메서드명에 `기본값`보다는 `현재값`과 같은 의미가 들어가야 할 것 같네요!

주디의 의견을 듣고보니 기본값은 항상 일정한 고정값으로의 의미가 강한것 같아서 현재값을 지정하는 의미를 넣기 위해`configureStepperCurrentValue` 로 수정해 주었습니다 !

Reply:
> `countToDouble`
> 
> [API Design Guidelines](https://www.swift.org/documentation/api-design-guidelines/)에서 다음과 같은 문구가 있습니다.
> 
> > Name variables, parameters, and associated types according to their roles, rather than their type constraints.
> > : 타입 제약 조건이 아닌 **역할**에 따라 변수, 매개 변수 및 관련 유형의 이름을 지정합니다.
> > ex) var string = "Hello" ---> var greeting = "Hello"
> 
> 이렇듯 특정 타입을 명시하기 보다는 해당 프로퍼티의 역할이나 의미로 명명하는게 좋겠습니다!

네이밍 컨벤션에 맞도록 stepper에 값을 할당하기위해 `stepperValue`라고 상수명을 변경하였습니다.